controller/server/server.go

```diff
@@ -62,7 +63,8 @@ func (s *Service) Run(ctx context.Context, rq *pb.RunRequest) (*pb.RunReply, err
 	if s.c == nil {
 		return nil, errors.New("nil engine client in controller server")
 	}
-	id, err := startJob(s.c, "models/three_layer.xml", "out")
+	
+	id, err := startJob(s.c, "models/barkley3d_coupling.xml", "out", rq)
```

Comment:
can you grab the model/out strings from the `RunRequest` instead of hardcoding? that'll make it easier to change later